tests/opds.t

```diff
@@ -49,6 +49,11 @@ $mojo->mock(
         my $tt = Template->new( { INCLUDE_PATH => $cwd . "/templates" } ) || die "$Template::ERROR\n";
         my $output;
 
+        # this mocks the Mojo url_for helper function, which is now
+        # used in the templates
+        $vars{c} = {
```

Comment:
Maybe one to put in mocks.pl?

Reply:
The mock for `url_for` is specific to this `render_to_string` mock, and needs to be in the same place.

It might be reasonable to remove this whole Mojo mock, including `render_to_string`, into mocks.pl, if we wanted to use it in some other test.